modules/preview-react/status-indicator/lib/StatusIndicator.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 
 import {ExtractProps, createContainer, createElemPropsHook} from '@workday/canvas-kit-react/common';
-import {HStack} from '@workday/canvas-kit-react/layout';
+import {Flex} from '@workday/canvas-kit-react/layout';
 import {StatusIndicatorIcon, statusIndicatorColors} from './StatusIndicatorIcon';
 import {StatusIndicatorLabel} from './StatusIndicatorLabel';
 import {useStatusIndicatorModel} from './hooks';
 
-export interface StatusIndicatorProps extends Partial<ExtractProps<typeof HStack, never>> {
+export interface StatusIndicatorProps extends Partial<ExtractProps<typeof Flex, never>> {
```

Comment:
I think this can just be `export interface StatusIndicatorProps extends Flex` i think the partial was to remove the need for `spacing` I'd double check